components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/bean/Scope.java

```diff
@@ -0,0 +1,93 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.carbon.identity.oauth2.bean;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class Scope implements Serializable {
+
+    /**
+     * Required
+     **/
+    String id;
+
+    /**
+     * Required
+     **/
+    String name;
+
+    /**
+     * Required
+     **/
+    String description;
+
+    /**
+     * Optional
+     **/
+    List<String> bindings;
+
+    public Scope() {
+
+    }
+
+    public Scope(String name, String description, List<String> bindings) {
```

Comment:
two custroctors for Scope(String name, String description, List<String> bindings) and Scope(String name, String description) and remove requires/optional comments